CONTRIBUTING.md

```diff
@@ -48,7 +48,10 @@ permission to use and redistribute your contributions as part of the project.
 Any substantial design deserves a design document. Design documents are written with Google Docs and
 should be shared with the community by adding the doc to our [Team Drive](https://drive.google.com/corp/drive/u/0/folders/0AIS5p3eW9BCtUk9PVA)
 and sending a note to the appropriate working group to let people know the doc is there. To get write access
-to the drive, you'll need to be an official contributor to the Istio project per GitHub.
+to the drive, you'll need to be a [member](ROLES.md#member) of the Istio organization.
+
+We have a common [template](https://docs.google.com/document/d/1cpolPNH_RtSobUjTkXyyxsCJT7EKD8fGiN2TgYLeJu8/edit#heading=h.7zgnj8bwqfld).
```

Comment:
We have a common design doc template.

Maybe [design doc template] as the link.